antimarkdown/nodes.py

```diff
@@ -11,7 +11,7 @@
 hex_color = collections.defaultdict(lambda: 'none')
 try:
     import mdx_hilite
-    hex_color.update((h, c) for c, h in mdx_hilite.color_hex.iteritems())
+    hex_color.update((h, c) for c, h in mdx_hilite.color_hex.items())
```

Comment:
We're dropping support for `mdx_hilite` so this whole business can probably be removed altogether.